datadog-monitors-downtimeschedule-handler/docs/README.md

```diff
@@ -0,0 +1,139 @@
+    <a href="#schedule" title="Schedule">Schedule</a>: <i><a href="schedule.md">Schedule</a></i>
+</pre>
+
+## Properties
+
+#### DisplayTimezone
+
+The timezone in which to display the downtime's start and end times in Datadog applications. This is not used as an offset for scheduling.
+
+_Required_: No
+
+_Type_: String
+
+_Update requires_: [No interruption](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-no-interrupt)
+
+#### Message
+
+A message to include with notifications for this downtime. Email notifications can be sent to specific users by using the same `@username` notation as events.
+
+_Required_: No
+
+_Type_: String
+
+_Update requires_: [No interruption](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-no-interrupt)
+
+#### MuteFirstRecoveryNotification
+
+If the first recovery notification during a downtime should be muted.
+
+_Required_: No
+
+_Type_: Boolean
+
+_Update requires_: [No interruption](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-no-interrupt)
+
+#### Scope
+
+The scope to which the downtime applies. Must follow the [common search syntax](https://docs.datadoghq.com/logs/explorer/search_syntax/).
+
+_Required_: Yes
+
+_Type_: String
+
+_Update requires_: [No interruption](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-no-interrupt)
+
+#### NotifyEndStates
+
+States that will trigger a monitor notification when the `notify_end_types` action occurs.
+
+_Required_: No
+
+_Type_: List of String
+
+_Update requires_: [No interruption](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-no-interrupt)
+
+#### NotifyEndTypes
+
+Actions that will trigger a monitor notification if the downtime is in the `notify_end_types` state.
+
+_Required_: No
+
+_Type_: List of String
+
+_Update requires_: [No interruption](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-no-interrupt)
+
+#### MonitorIdentifier
+
+_Required_: Yes
+
+_Type_: <a href="monitoridentifier.md">MonitorIdentifier</a>
+
+_Update requires_: [No interruption](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-no-interrupt)
+
+#### Schedule
+
+A recurring downtime schedule definition.
+
+_Required_: No
+
+_Type_: <a href="schedule.md">Schedule</a>
+
+_Update requires_: [No interruption](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-no-interrupt)
+
+## Return Values
+
+### Ref
+
+When you pass the logical ID of this resource to the intrinsic `Ref` function, Ref returns the Id.
```

Comment:
```suggestion
When you pass the logical ID of this resource to the intrinsic `Ref` function, `Ref` returns the ID.
```

Reply:
This is autogenerated as well, so we do not have control over it